source/docs/software/networktables/creating-sendable-classes.rst

```diff
@@ -0,0 +1,124 @@
+.. _creating-sendable-classes:
+
+
+Using the Sendable interface
+============================
+
+What is Sendable?
+-----------------
+
+Sendable is the base class that allows something to be sent over NetworkTables for debugging through programs such as OutlineViewer, ShuffleBoard and SmartDashboard. Base WPI classes such as Command and Subsystem, as well as actuator classes such as DoubleSolenoid, implement the Sendable interface. The WPI documentation of the Sendable interface, as well as a full list of WPI classes which implement Sendable, can be found on the WPI docs for Sendable (`Java <https://first.wpi.edu/FRC/roborio/release/docs/java/edu/wpi/first/wpilibj/Sendable.html>`__, `C++
+<https://first.wpi.edu/FRC/roborio/release/docs/cpp/classSendable.html>`__).
+
+Putting a Sendable on NetworkTables
+-----------------------------------
+
+Adding the current state of a Sendable class to NetworkTables is as simple as adding it's entry using the putData() method.
+
+.. tabs::
+
+    .. code-tab:: c++
+
+        frc::SmartDashboard::PutData("my sendable", m_SendableClass)
+
+        frc::ShuffleboardTab& m_tab = frc::Shuffleboard::GetTab("tabName");
+        m_tab.Add("Motor", m_sendableClass);
+
+    .. code-tab:: java
+
+        SmartDashboard.putData("my sendable", sendable)
+
+        ShuffleboardTab m_tab = Shuffleboard.getTab("name")
+        m_tab.add(["my sendable",] sendable);
```

Comment:
It'd be clearer to have two examples here, one for using an explicit name and one using the sendable's name

The Java and C++ examples should be as close as possible (eg same tab name for shuffleboard, same variable names, etc)